nurseconnect/templates/base.html

```diff
@@ -93,25 +93,18 @@
         {% wagtailuserbar %}
 
         <main role="main">
-            {% include "core/header.html" %}
+            <div class="Logo--auth icon-logo"></div>
```

Comment:
This doesn't look right. The convention should be `Logo Logo--auth`, so a block with a block modifier.